From: Felix Fietkau <nbd@nbd.name>
Date: Mon, 27 Oct 2014 00:00:00 +0100
Subject: [PATCH] mac80211: preseve AP mode keys across STA reconnect

Used for AP+STA support in OpenWrt - preserve AP mode keys across STA reconnect
---
 net/mac80211/cfg.c | 1 -
 1 file changed, 1 deletion(-)

--- a/net/mac80211/cfg.c
+++ b/net/mac80211/cfg.c
@@ -1635,7 +1635,6 @@ static int ieee80211_stop_ap(struct wiph
 	link_conf->bssid_indicator = 0;
 
 	__sta_info_flush(sdata, true);
-	ieee80211_free_keys(sdata, true);
 
 	link_conf->enable_beacon = false;
 	sdata->beacon_rate_set = false;
